test/helpers/cilium-cli.go

```diff
@@ -0,0 +1,43 @@
+package helpers
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/cilium/cilium/test/config"
+)
+
+const (
+	CiliumCliCmd = "cilium"
+)
+
+type CiliumCli struct {
+	executor Executor
+	log      *logrus.Entry
+}
+
+func CreateCiliumCli(log *logrus.Entry) (c *CiliumCli) {
+	var environ []string
+	if config.CiliumTestConfig.PassCLIEnvironment {
+		environ = append(environ, os.Environ()...)
+	}
```

Comment:
It'd be good to have godocs on the exported types in this PR because we are adding a new structure to the codebase which we will invest heavily in in the future so let's make sure things are documented so that we can easily reuse them going forward.